pkgs/applications/window-managers/river/default.nix

```diff
@@ -21,16 +21,17 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "river";
-  version = "0.2.6";
+  version = "0.3.0";
 
   outputs = [ "out" ] ++ lib.optionals withManpages [ "man" ];
 
-  src = fetchFromGitHub {
-    owner = "riverwm";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "river";
     repo = "river";
-    rev = "refs/tags/v${finalAttrs.version}";
```

Comment:
Does the Gitea fetcher not support this ref format?

Reply:
I suspected that it was not, but I was wrong. I added it back and the derivation is not affected.